### PR TITLE
docs: changes in documentation

### PR DESCRIPTION
**Pull request by harshalini**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+}
+.nav-pages a {
+    border-bottom: solid 2px var(--white);
+}
+.nav-pages a:hover {
+    opacity: 50%;
+}
+header {
+    display: flex;
+    margin: 100px 25px 0px 25px;
```
Comment (Raghav888) [inline style.css:55]: Use rem only if more than 10px.

**Review by Raghav888 — CHANGES_REQUESTED** [review]
Use rem , rest looks good

**Review by Manju-sree — COMMENTED** [review]
looks pretty , try to add unique class-names with the name you choose for component library..

**Comment by tushartiwari7** [comment]
Code is good! 
But from next time try to make several small PR's instead of one huge PR.

**Comment by harshalini** [comment]
> 

Yes, thanks for suggestion.

**Pull request merged**